README.md

```diff
@@ -136,7 +137,8 @@ http://numbersapi.com/random?min=10&max=20
 13 is the number of provinces and territories in Canada.
 </pre>
 
-<h3 id="json">Json</h3>
```

Comment:
In addition to Matt's comments about preserving the IDs by reverting back to HTML elements, I think all that's needed here is a linebreak right here.

Reply:
Hmm. From running this locally on my machine, the ### seems to work with the IDs still, so from my most updated version, there is a line break there. Good looking out!